### PR TITLE
Quicviz yg

### PR DESCRIPTION
**Pull request by yueyaog**
QUICviz WDL that is used for PECGS CNV analysis

### CODE REVIEW

File: PECGS-QUICviz/QUICviz.wdl [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+        ls outputs/
+        readlink -f outputs/*png
+    >>>
+    output {
+        Array[File] plot = glob("outputs/*.png")
+    }
+    runtime {
+        docker: quicvizDocker
+        memory: "16 GB"
```
Comment (lfleming1022) [inline PECGS-QUICviz/QUICviz.wdl:81]: Do we want to make memory a variable?

File: PECGS-QUICviz/QUICviz.wdl [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+            --tumor_type ~{tumorType} \
+            --normal_acf ~{allelicCountsNormal} \
+            --normal_cr ~{denoisedCopyRatiosNormal} \
+            --tumor_acf ~{allelicCountsTumor} \
+            --tumor_cr ~{denoisedCopyRatiosTumor} \
+            --tumor_cr_seg ~{calledCopyRatioSegTumor} \
+            --tumor_seg_oncotated ~{oncotatedCalledTumor} \
+            --output_dir outputs/
+
+        ls outputs/
```
Comment (yueyaog) [inline PECGS-QUICviz/QUICviz.wdl:73]: Remove this line

File: PECGS-QUICviz/QUICviz.wdl [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+task mergeImages {
+    input {
+        String SampleID
+        String TumorType
+        Array[File] plot
+    }
+    command <<<
+        mkdir -p output/images
+        for i in `ls ~{sep=" " plot}`; do mv $i output/images/; done
+        ls output/images/
```
Comment (yueyaog) [inline PECGS-QUICviz/QUICviz.wdl:95]: remove this line

File: PECGS-QUICviz/QUICviz.wdl [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+        source activate NeoVax-Input-Parser
+        pip3 install img2pdf
```
Comment (yueyaog) [inline PECGS-QUICviz/QUICviz.wdl:97]: install this in QUICviz docker

**Review by yueyaog — COMMENTED** [review]
Hackathon Apr 26

**Review by lfleming1022 — APPROVED** [review]

**Pull request merged**